qemu/tests/cfg/multi_nics_hotplug.cfg

```diff
@@ -22,6 +22,8 @@
                 pci_model = virtio
             nics = ""
             extra_params += "-net none"
+            s390x:
+                pci_model = virtio-net-ccw
```

Comment:
As discussed at VT, ccw devices are not pci devices, so `pci_model` is not good. If you don't want to replace it in cfg and `nic_hotplug`, please add a comment or FIXME.

Reply:
add comment for it